homeassistant/components/sensor/glances.py

```diff
@@ -64,11 +66,9 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     try:
         response = requests.get(url, timeout=10)
         if not response.ok:
-            _LOGGER.error("Response status is '%s'", response.status_code)
-            return False
+            _LOGGER.warn("Response status is '%s'", response.status_code)
```

Comment:
I opted for showing errors during component setup, which allows for troubleshooting when configuring the component, and showing only a debug message, so the log doesn't get polluted if the target host is down (ie. laptops). I hope it makes sense O:-]